requirements.txt

```diff
@@ -16,3 +16,4 @@ requests
 plotly
 torch
 torchvision
+dash-dangerously-set-inner-html
```

Comment:
Ah yeah, we should add that to the `environment.yml` as well at a later time, if we really need it.

Reply:
We should actually really add it, before considering that as resolve.